pyproject.toml

```
@@ -1,4 +1,67 @@
-# black project prefers pyproject.toml
-# that's why we have this file in addition to other setting files
+[build-system]
+requires = ["setuptools", "pytest-runner", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "slack_sdk"
+dynamic = ["version", "readme"]
+description = "The Slack API Platform SDK for Python"
+license = { text = "MIT" }
+authors = [{ name = "Slack Technologies, LLC", email = "opensource@slack.com" }]
+requires-python = ">=3.6"
+keywords = [
+	"slack",
+	"slack-api",
+	"web-api",
+	"slack-rtm",
+	"websocket",
+	"chat",
+	"chatbot",
+	"chatops",
+]
+classifiers = [
+	"Development Status :: 5 - Production/Stable",
+	"Intended Audience :: Developers",
+	"Topic :: Communications :: Chat",
+	"Topic :: System :: Networking",
+	"Topic :: Office/Business",
+	"License :: OSI Approved :: MIT License",
+	"Programming Language :: Python",
+	"Programming Language :: Python :: 3",
+	"Programming Language :: Python :: 3.6",
+	"Programming Language :: Python :: 3.7",
+	"Programming Language :: Python :: 3.8",
+	"Programming Language :: Python :: 3.9",
+	"Programming Language :: Python :: 3.10",
+	"Programming Language :: Python :: 3.11",
+	"Programming Language :: Python :: 3.12",
+]
+
+
+[project.urls]
+homepage = "https://github.com/slackapi/python-slack-sdk"
+
+[tool.setuptools.packages.find]
+include = ["slack_bolt*", "slack*"]
```

I'm not 100% sure, but it feels like there shouldn't be `slack_bolt_*`.
```suggestion
include = ["slack*"]
```
Also, I assume `slack*` should cover `slack_sdk*` which is necessary.